src/main/java/com/sumologic/jenkins/jenkinssumologicplugin/sender/SumoLogicFileUploadStep.java

```diff
@@ -214,15 +283,17 @@ public void sendFilesData(File localFile) throws IOException {
         private static final LogSenderHelper logSenderHelper = LogSenderHelper.getInstance();
         private final String url;
         private final String sourceCategory;
-        private final String jobName;
-        private final int buildNumber;
+        private final String hostName;
+        private final String sourceName;
+        private final HashMap<String, String> fields;
 
-        FileListUploader(List<File> fileList, String url, String sourceCategory, String jobName, int buildNumber) {
+        FileListUploader(List<File> fileList, String url, String sourceCategory, String hostName, String sourceName, HashMap<String, String> fields) {
```

Comment:
These are not backward compatible changes, are these functions just internal?

Reply:
Function is internal and used by sumoupload function